db/migrate/20190705173948_renumber_group_visibility_levels.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class RenumberGroupVisibilityLevels < ActiveRecord::Migration[5.2]
+  def up
+    execute "UPDATE groups SET visibility_level = 4 WHERE visibility_level = 3"
```

Comment:
Super minor but this could have been `SET visibility_level = visibility_level + 1` right? And below the opposite? (`-1`).
No need to change it just an observation :)

Reply:
Yeah, that would have been better (I would need to modify it so that 0-level groups shouldn't be bumped to level 1 on `up`).